src/ui/types/index.d.ts

```diff
@@ -85,6 +85,10 @@ import {
 import { InsuranceFeedbackVariables } from './feedback';
 import { SanitiseValueObjParams } from './sanitise-value';
 
+type FieldsWithValues = {
+  [key: string]: string;
+};
```

Comment:
Could we use `ObjectType` instead? Few instances of `ObjectType` declared, will be a nice and quick DRY change 😎

Reply:
Can we move this into its own file like the other types?

Happy to do after this is merged